exec/java-exec/pom.xml

```diff
@@ -410,6 +410,11 @@
       <artifactId>joda-time</artifactId>
       <version>2.9</version>
     </dependency>
+    <dependency>
+      <groupId>com.clearspring.analytics</groupId>
+      <artifactId>stream</artifactId>
+      <version>2.7.0</version>
+    </dependency>
```

Comment:
This library must be excluded from the Drill JDBC pom. (Yes, a real nuisance...)